classes/Module/ModuleDisabler.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License (AFL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\AutoUpgrade\Module;
+
+use Db;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Class ModuleDisabler is used to disable/enable modules, there are two ways of
+ * disabling a module:
+ *  - move its whole folder somewhere else
+ *  - update the database to set its active column to 0
+ */
+class ModuleDisabler
```

Comment:
We are already in the Module namespace, it's redundant.
```suggestion
class Disabler
```

Reply:
Ok, but then we keep the `Module` namespace? not Addons?

Reply:
Depends on the context.
If this class has for aim to disable a module, keep the `Module` namespace